api/src/bigquerytest/java/org/pmiops/workbench/api/CohortReviewControllerBQTest.java

```diff
@@ -1463,4 +1424,15 @@ private void stubMockFirecloudGetWorkspace() throws ApiException {
     workspaceResponse.setAccessLevel(WorkspaceAccessLevel.WRITER.toString());
     when(mockFireCloudService.getWorkspace(NAMESPACE, NAME)).thenReturn(workspaceResponse);
   }
+
+  private void stubMockFirecloudGetWorkspaceAcl() throws ApiException {
+    WorkspaceACL workspaceAccessLevelResponse = new WorkspaceACL();
```

Comment:
nit: I generally like to make local vars final where possible.  Also, "stub mock" seems a bit redundant, or at least a mouthful. 

Can you use `ImmutableMap.of()` to make the acl map? I'd also name it to indicate the key and value, like `userEmailToAccessEntry`.

Finally, I'm surprised you don't need more than one access level response for the tests, but i haven't read closely. You might think about having a function to build that response object based on passed-in info, and the use different responses in different `when` methods to test things.

Reply:
"Stub Mock" is what is used already in the file, so I'm inclined to stick with existing file naming. It is somewhat redundant. 

Currently, one access level is sufficient, and I think makes a good default. You can always override this for specific tests. (These are mostly for the sake of setup functions)

Reply:
That file is the only place that uses it. I'd vote for just changing it to `mock`

Do we have test cases for when the access level is not sufficient? That's why I was curious this function always gave one level of access. If that's a follow-on story that's OK too.